src/elements/Button/Button.js

```diff
@@ -229,18 +231,23 @@ class Button extends Component {
       useKeyOnly(toggle, 'toggle'),
       useKeyOrValueAndKey(animated, 'animated'),
       useKeyOrValueAndKey(attached, 'attached'),
+    )
+    const labeledClasses = cx(
+      useKeyOrValueAndKey(labelPosition || !!label, 'labeled'),
+    )
+    const wrapperClasses = cx(
+      useKeyOnly(disabled, 'disabled'),
       useValueAndKey(floated, 'floated'),
     )
-    const wrapperClasses = cx(useKeyOnly(disabled, 'disabled'))
+
     const rest = getUnhandledProps(Button, this.props)
-    const ElementType = getElementType(Button, this.props, () => {
-      if (!_.isNil(label) || !_.isNil(attached)) return 'div'
-    })
```

Comment:
I've moved this inline function to method, so it will not be created on each render.